docs/credits.md

```diff
@@ -961,6 +962,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | Octopus.Server.Extensibility.IssueTracker.AzureDevOps | Octopus Deploy                                                            | [https://github.com/OctopusDeploy/AzureDevOpsIssueTracker](https://github.com/OctopusDeploy/AzureDevOpsIssueTracker)                                                                                                        | [Apache-2.0](https://github.com/OctopusDeploy/AzureDevOpsIssueTracker/blob/master/LICENSE.txt)                      |
 | Octopus.Server.Extensibility.IssueTracker.GitHub  | Octopus Deploy                                                            | [https://github.com/OctopusDeploy/GitHubIssueTracker](https://github.com/OctopusDeploy/GitHubIssueTracker)                                                                                                                  | [Apache-2.0](https://github.com/OctopusDeploy/GitHubIssueTracker/blob/master/LICENSE.txt)                           |
 | Octopus.Server.Extensibility.JiraIntegration      | Octopus Deploy                                                            | [https://github.com/OctopusDeploy/JiraIntegration](https://github.com/OctopusDeploy/JiraIntegration)                                                                                                                        | [Apache-2.0](https://github.com/OctopusDeploy/JiraIntegration/blob/master/LICENSE.txt)                              |
+| Octopus.Server.Portal                             | Octopus Deploy                                                            | [https://octopus.com/](https://octopus.com/)                                                                                                                                                                                | [Apache-2.0](https://licenses.nuget.org/Apache-2.0)                                                                 |
```

Comment:
@matt-richardson I thought we were going to point to the GitHub repo?

Reply:
We do for the [repository](https://github.com/OctopusDeploy/OctopusDeploy/blob/master/newportal/Octopus.Server.Portal.nuspec#L17), but this shows the [projectUrl](https://github.com/OctopusDeploy/OctopusDeploy/blob/master/newportal/Octopus.Server.Portal.nuspec#L15), it appears.